### PR TITLE
Add completion for stats.

### PR DESCRIPTION
**Pull request by jessfraz**
Docker-DCO-1.1-Signed-off-by: Jessica Frazelle jess@docker.com (github: jfrazelle)

### CODE REVIEW

File: contrib/completion/bash/docker [hunk 1]
```diff
@@ -789,6 +789,13 @@ _docker_start() {
 	esac
 }
 
+_docker_stats() {
+	local counter=$(__docker_pos_first_nonflag)
+	if [ $cword -eq $counter ]; then
+		__docker_containers_running
```
Comment (tianon) [inline contrib/completion/bash/docker]: Doesn't `docker stats` take any number of containers as arguments? :wink:

Comment (jfrazelle) [inline contrib/completion/bash/docker]: updated just some monday morning copy pasta

**Comment by tianon** [comment]
LGTM, but has pointed out some bizarre behavior I'm seeing (ie, `docker stats non_running_container` hangs, and I can't see any reason for that client code to be hanging)

**Comment by tianon** [comment]
(as opposed to `docker stats not_a_container`, which comes back immediately with an error)

**Comment by jfrazelle** [comment]
@crosbymichael this is because it is waiting for the container to start right?

**Comment by jfrazelle** [comment]
I think it is intentional

**Comment by tianon** [comment]
So the stats wait for _all_ the containers to start before they display?  What if a container stops while I'm viewing stats on it?  My dashboard just freezes up? :frowning:

**Comment by icecrime** [comment]
I'm pretty sure the ability to call `docker stats` on a stopped container is intended, and yes metrics will start to broadcast when the container starts. I just tested on multiple containers, and indeed this is very weird but stats only start showing up when every container is running...

**Comment by jfrazelle** [comment]
Regardless that shouldn't be a blocker for this :P

**Comment by jfrazelle** [comment]
This would be a nice to have for 1.5 ;)

**Comment by icecrime** [comment]
Agreed: LGTM.

**Pull request merged**